### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by DoraeM2112**
readability↑

### CODE REVIEW

**Comment by niv** [comment]
Thanks for the contrib!

This is from internal tooling that prepares the squashed commit for Github. It already should have caught these repeated newlines .. we're looking into it.

**Comment by niv** [comment]
Tracked internally: NV-3643422

**Comment by asajaroff** [comment]
Hugh PR! thanks

**Review by asajaroff — APPROVED** [review]

File: kernel-open/nvidia/nv-i2c.c [hunk 1]
```diff
@@ -273,246 +273,6 @@ void NV_API_CALL nv_i2c_del_adapter(nv_state_t *nv, void *data)
...
-
-
-
-
-
-
-
-
-
-
```
Comment (MattTheCuber) [inline kernel-open/nvidia/nv-i2c.c:322]: this line shouldnt be hear, it will improve readability

**Comment by PAR2020** [comment]
@DoraeM2112, can you please execute the CLA for this PR? Thank you.

**Pull request closed without merging**